src/lib/exposes.ts

```diff
@@ -615,6 +615,9 @@ export const presets = {
     // Specific
     ac_frequency: () => new Numeric('ac_frequency', access.STATE).withLabel('AC frequency').withUnit('Hz').withDescription('Measured electrical AC frequency'),
     action: (values: string[]) => new Enum('action', access.STATE, values).withDescription('Triggered action (e.g. a button click)'),
+    action_type: (values: string[]) => new Enum('action_type', access.STATE, values).withDescription('Type of the rotation, value in the first message is `step` and in the next messages value is `rotate`'),
```

Comment:
It is not needed to put them here as these are not generic, so in the definition `exposes` use e.g. 

```js
e.enum('action_type', access.STATE, ['step', 'rotate']).withDescription('Type of the rotation, value in the first message is `step` and in the next messages value is `rotate`')
```

Reply:
Taken care of